widgy/compat.py

```diff
@@ -0,0 +1,20 @@
+import django
+
+
+def check_if_field_name_exists_on_cls(field_name, cls):
+    """
+    Check if a field is already present on a model in a `safe` manner that does
+    not trigger or violate application loading.
+    """
+    if django.get_version() < "1.7":
```

Comment:
Cool, I was sort of remembering that there was a case where this didn't work but I couldn't figure out what it was.